newdle/templates/joined_email.html

```diff
@@ -0,0 +1,9 @@
+<body>
+  <p>Hi,</p>
+  <p>
+    <strong>{{ participant }}</strong> has joined to "<strong>{{ title }}</strong>".
+  </p>
+  <p>
+    You can get more info using your <a href="{{ answer_link }}"> personal link</a>.
```

Comment:
I don't think including the user's answer link makes sense here.

Better link to the newdle summary instead. Any maybe we could even include the user's availability information in this email?